lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -7,6 +7,7 @@
 module ApplianceConsole
   CERT_DIR = ENV['KEY_ROOT'] || RAILS_ROOT.join("certs")
   KEY_FILE = "#{CERT_DIR}/v2_key"
+  KEY_FILE_BACKUP = "#{CERT_DIR}/vk.bak"
```

Comment:
OK. I'll rename it. Should I also add a freeze to KEY_FILE. It's not suggested by travis but rubymine.